skbase/tests/test_exceptions.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+"""Tests for skbase exceptions.
+
+tests in this module:
+
+    test_exceptions_raise_error - Test that skbase exceptions raise expected error.
+"""
+from typing import List
+
+import pytest
+
+from skbase._exceptions import FixtureGenerationError, NotFittedError
+
+__author__: List[str] = ["fkiraly", "RNKuhns"]
```

Comment:
I don't think I should be an author? I did not write the test or conceive of it.